backend/api.py

```diff
@@ -348,10 +350,27 @@ def _load(self):
             self.stats_image_map_id = stats_data['image_map_id']
         else:
             self.has_stats = False
-    
+
     def key_scenes(self):
         return list( KeyScene.Query(self.redis_conn, self.mission_name).act_number(self.number).items() )
-    
+
+    def banner_styles(self):
+        """
+        Returns CSS to be applied to the act's banner, to apply styles defined
+        in the mission's _meta file.
+        """
+
+        styles = []
+        if self.banner_colour:
+            styles.append('background-color: %s' % self.banner_colour)
+        if self.banner_background:
+            styles.append('background-image: url(%s%s/images/banners/%s)' % (
+                settings.MISSIONS_STATIC_URL,
```

Comment:
A note that we should probably move to managing static assets properly. I've created #229 to track this.